lib/api/resources.ts

```diff
@@ -580,6 +587,23 @@ export const RESOURCES = {
     filterFields: [],
   },
 
+  // NOVES-FI
+  noves_transaction: {
+    path: '/api/v2/proxy/noves-fi/transactions/:hash',
+    pathParams: [ 'hash' as const ],
+  },
+  noves_address_history: {
+    path: '/api/v2/proxy/noves-fi/addresses/:address/transactions',
+    pathParams: [ 'address' as const ],
+    filterFields: [],
+  },
+  noves_describe_txs: {
+    path: '/api/v2/proxy/noves-fi/transactions',
+    queryParams: [
```

Comment:
We typically do not describe all query parameters for resources due to their potential abundance. Is it necessary? I cannot find any other instance in the code where it is utilized.